extensions/data-workspace/src/dialogs/newProjectDialog.ts

```diff
@@ -140,16 +148,7 @@ export class NewProjectDialog extends DialogBase {
 
 			// update hover text if a new workspace will be created for this project
 			if (!vscode.workspace.workspaceFile) {
-				workspaceTextBox.placeHolder = path.join(this.model.location, `${this.model.name}.code-workspace`);
-			}
-		}));
-
-		this.register(projectNameTextBox.onTextChanged(() => {
-			this.model.name = projectNameTextBox.value!;
-
-			// update hover text if a new workspace will be created for this project
-			if (!vscode.workspace.workspaceFile) {
-				workspaceTextBox.placeHolder = path.join(this.model.location, this.model.name, `${this.model.name}.code-workspace`);
+				workspaceTextBox.updateProperty('title', path.join(this.model.location, `${this.model.name}.code-workspace`));
```

Comment:
workspaceTextBox.title not working for you?

Reply:
Yeah it wasn't working because I missed updating a file. Sent out a PR to fix it https://github.com/microsoft/azuredatastudio/pull/13095